### PR TITLE
hw 2 Storybook

### PR DESCRIPTION
**Pull request by vbond**

### CODE REVIEW

File: src/components/EmptyResults.vue [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<template>
+  
+  <div class="col text-center no-found">
+    <h2 class="text-light font-weight-light">
+      No films found
+    </h2>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
```
Comment (ghost) [inline src/components/EmptyResults.vue]: move in global styles, not nessesary imprort in every component

also vue doesn't like style script inside
```
<div id="vue-app">
   // bad bad 
   <style
   <script
</div>
```

File: src/components/EmptyResults.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<template>
```
Comment (ghost) [inline src/components/EmptyResults.vue]: if your component doesn't do anything please use functional component
https://vuejs.org/v2/guide/render-function.html#Passing-Attributes-and-Events-to-Child-Elements-Components
https://www.youtube.com/watch?v=7lpemgMhi0k

File: src/components/EmptyResults.vue [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+<script>
+
+export default {
+  name: 'EmptyResults'
+};
+
+</script>
+
+<style scoped>
```
Comment (ghost) [inline src/components/EmptyResults.vue]: in shared components not always useful use scoped styles
https://www.youtube.com/watch?v=x78G5U2C3zk epamer guy

File: src/components/LogoFooter.vue [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <footer class=" fixed-bottom row justify-content-md-center">
```
Comment (ghost) [inline src/components/LogoFooter.vue:2]: whitespace, configure formatter if you wish :)

File: src/components/LogoFooter.vue [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+      <div class="footer col-md-8 justify-content-md-center">
+        <div class="col col-md-auto text-center">
+          <img src="../assets/movielogo.png" class="logo" alt="Netflix Roulette logo">
```
Comment (ghost) [inline src/components/LogoFooter.vue:5]: make global configuration for assests folder, in many cases those folder can be changed
or webpack(define-webpack-plugin) or window object(not preferable but easier)

Comment (vbond) [reply src/components/LogoFooter.vue:5]: could not find how to do this, solutions I found  are not working.. could you give the link to working example?

**Pull request merged**